.gitignore

```diff
@@ -67,6 +67,7 @@ nosetests.xml
 coverage.xml
 *,cover
 .hypothesis/
+tests/acceptance/login_cookies.pkl
```

Comment:
This is not really a coverage report, so maybe we should add an extra section for `# Application specific ignores` or something like that.